terraform/cloudfoundry/acl_bosh.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_network_acl_rule" "110_bosh" {
+    network_acl_id = "${aws_network_acl.cf.id}"
+    protocol = "tcp"
+    rule_number = 110
+    rule_action = "allow"
+    from_port = 0
+    to_port = 65535
+    cidr_block = "10.0.0.6/32"
```

Comment:
Ditto the comment in https://github.com/alphagov/paas-cf/pull/102/files#r52454687 - we have already have a variable for this: https://github.com/alphagov/paas-cf/blob/7a95a56745577c580ca189ac380d528da17ed82a/terraform/globals.tf#L79-L82